internal/ingress/controller/controller.go

```diff
@@ -91,6 +91,7 @@ type Configuration struct {
 	UpdateStatus           bool
 	UseNodeInternalIP      bool
 	ElectionID             string
+	ElectionTTL            int
```

Comment:
Can we make it a duration here so the actual time unit isn't predefined?

Reply:
I'm not sure if it's a good idea at this moment, since the leader election was already defined in seconds :/

Reply:
Well, the actual `ttl` used internally can stay an integer for nanoseconds, but the value we hand from arguments should be a duration so one can hand `30m`, `1h`, `45s` or whatsoever.

`time.Duration` is just an `int64` representing nanoseconds. So you could omit multiplying it by `time.Second` then.